rafiki/model/dataset.py

```diff
@@ -13,6 +13,7 @@
 import abc
 import tempfile
 import csv
+import pandas as pd
```

Comment:
You'll need to add this dependency to [`rafiki/requirements.txt`](https://github.com/nginyc/rafiki/blob/master/rafiki/requirements.txt). `pandas` won't be available in the Rafiki's runtime environment.